lib/honeybadger/plugins/delayed_job/plugin.rb

```diff
@@ -8,9 +8,17 @@ class Plugin < ::Delayed::Plugin
         callbacks do |lifecycle|
           lifecycle.around(:invoke_job) do |job, &block|
             begin
-              begin #buildin suport for Delayed::PerformableMethod
-                component = job.payload_object.object.is_a?(Class) ? job.payload_object.object.name : job.payload_object.object.class.name
-                action    = job.payload_object.method_name.to_s
+
+              begin
+                if job.payload_object.class == ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper
```

Comment:
@pkorros won't this blow up in < Rails 4.2 which will always cause the fallback on line 22 to be used?